recipes/mosaic/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "mosaic" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/E3SM-Project/mosaic/archive/{{ version }}.tar.gz
+  sha256: 6393c3bc58257c1214903e30f19cdf7688ba6262215088655e77d9c1e852d57e
+
+build:
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - cartopy
+    - cmocean
+    - h5netcdf
+    - netcdf4
+    - matplotlib-base
+    - numpy
+    - pyproj
+    - scipy
+    - xarray
+
+test:
+  imports:
+    - mosaic
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://github.com/E3SM-Project/mosaic/
+  summary: Graphics package for MPAS meshes
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  doc_url: https://github.com/E3SM-Project/mosaic/README.md
+  dev_url: https://github.com/E3SM-Project/mosaic/
+
+extra:
+  recipe-maintainers:
+    - andrewdnolan
```

Comment:
@andrewdnolan, please confirm that you are willing to be a maintainer.

Reply:
Also, please take a look at the recipe and make sure you don't see obvious mistakes.

Reply:
Yes, I'm happy to be a maintainer.

Everything seems correct to me, at least as far as I know. Thanks for spearheading this!

Reply:
Great, hopefully the linter will notice...